tests/unit/phpunit.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<phpunit bootstrap="./tests/bootstrap.php"
+<phpunit bootstrap="bootstrap.php"
 		 verbose="true"
 		 timeoutForSmallTests="900"
 		 timeoutForMediumTests="900"
 		 timeoutForLargeTests="900"
 >
-	<testsuite name='testing-unit'>
-		<directory suffix='Test.php'>./tests/unit</directory>
+	<testsuite name='unit'>
+		<directory suffix='Test.php'>.</directory>
 	</testsuite>
 	<!-- filters for code coverage -->
 	<filter>
-		<whitelist>
-			<directory suffix=".php">./appinfo</directory>
-			<directory suffix=".php">./lib</directory>
-		</whitelist>
+		<directory suffix=".php">../../twofactor_totp</directory>
+		<exclude>
+			<directory suffix=".php">../../../twofactor_totp/l10n</directory>
+			<directory suffix=".php">../../../twofactor_totp/tests</directory>
+		</exclude>
 	</filter>
 	<logging>
 		<!-- and this is where your report will be written -->
-		<log type="coverage-clover" target="./tests/output/clover-unit.xml"/>
+		<log type="coverage-clover" target="./clover.xml"/>
```

Comment:
we try to put all clover output into tests/output and also try to move all phpunit.xml into the repo root

Reply:
OK, I will adjust. I was looking at password_policy for clues.

Reply:
It's all mixed .. as usual :man_shrugging:

Reply:
PR #78